morerss/jike.py

```diff
@@ -61,8 +56,9 @@ def post2rss(data_plan, post):
       topic = post['topic']
       description += "来自圈子：<a href='https://m.okjike.com/topics/%s' target='_blank'>%s</a>" % (topic['id'], topic['content'])
 
+  import re
   item = PyRSS2Gen.RSSItem(
-    title = title,
+    title=re.split(',|，|\.|。|;|；|!|！|\?|？|~|<br/>', content)[0],
```

Comment:
Why not use `[...]`?

Reply:
Sorry, I don't quite understand, could you elaborate?

Reply:
Using [character classes](https://www.regular-expressions.info/charclass.html) (`[abc]`) instead of alteration (`a|b|c`) will make the regex easier to read.